session/manager.go

```diff
@@ -41,6 +42,28 @@ func (e *ErrAALNotSatisfied) EnhanceJSONError() interface{} {
 	return e
 }
 
+func (e *ErrAALNotSatisfied) PassReturnToParameter(requestURL string) error {
```

Comment:
Could you please add a dedicated, small test for this? :)

Reply:
OK! I've added.